.github/workflows/build-and-publish.yml

```diff
@@ -0,0 +1,28 @@
+name: publish
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [ 10.x, 12.x, 14.x ]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          registry-url: https://registry.npmjs.org/
+      - run: npm ci
+      - run: npm run test:ci
+      - run: npx semantic-release
```

Comment:
I'd suggest adding an `if` here to make sure only one of the builds does the publishing.

For example:
```
- name: 'Release'
  run: npx semantic-release
  if: github.ref == 'refs/heads/master' && matrix.node-version == '14.x'
```

Reply:
Absolutely, I forgot that we have multiple parallel jobs.